uaclient/security_status.py

```diff
@@ -120,27 +125,37 @@ def security_status(cfg: UAConfig) -> Dict[str, Any]:
     installed_packages = [package for package in cache if package.is_installed]
     summary["num_installed_packages"] = len(installed_packages)
 
-    security_upgradable_versions = filter_security_updates(installed_packages)
+    package_count = {"esm-infra": 0, "esm-apps": 0}
+    update_count = {"esm-infra": 0, "esm-apps": 0, "standard-security": 0}
 
-    package_count = {"esm-infra": 0, "esm-apps": 0, "standard-security": 0}
+    for package in installed_packages:
+        if is_esm_package(package, "esm-apps"):
+            package_count["esm-apps"] += 1
+        if is_esm_package(package, "esm-infra"):
+            package_count["esm-infra"] += 1
+
+    security_upgradable_versions = filter_security_updates(installed_packages)
```

Comment:
Just to double check, if we have a package with higher version on both `security` and `esm-infra` we will show them both on the json response right ?

Reply:
Yes, we will - they will be listed as separate updates. The only thing to sort is between ESM though, because it is guaranteed that we will not have the same version in `security` and any other ESM pocket

Reply:
Solved with the above comment on versions